x-pack/plugins/fleet/public/applications/integrations/sections/epm/constants.tsx

```diff
@@ -66,10 +66,10 @@ export const AssetTitleMap: Record<
     }
   ),
   'ml-module': i18n.translate('xpack.fleet.epm.assetTitles.mlModules', {
-    defaultMessage: 'ML modules',
+    defaultMessage: 'Anomaly Detection configurations',
   }),
   ml_module: i18n.translate('xpack.fleet.epm.assetTitles.mlModules', {
-    defaultMessage: 'ML modules',
+    defaultMessage: 'Anomaly Detection configurations',
```

Comment:
Nit. As above.

```suggestion
    defaultMessage: 'Anomaly detection configurations',
```

Reply:
Updated in f4cde98229ffc604f8b039d8595c396a70637b20